### DISCUSSION TITLE
为什么Vercel上部署后，设置ACCESS_CODE为环境变量没有生效呢？

### DISCUSSION THREAD

**Discussion by Aryous** [lobehub/lobehub#4667] [Self Hosting]
点击一键部署，设置了ACCESS_CODE和OPENAI_API_KEY的环境变量，部署过程中发现变量可以编辑，点击编辑后，部署被终止。于是我选择重新从自己的仓库导入部署，同样设置变量后，部署成功。但是发现ACCESS_CODE未生效。并且我可以设置Openrouter API。

**Comment by arvinxx** [comment]
你要发个请求看看
